test/controllers/util/FormFieldSpec.scala

```diff
@@ -117,140 +117,174 @@ class FormFieldSpec extends AnyWordSpec with MockitoSugar with BeforeAndAfterEac
         year = InputNameAndValue("Year", "2015", "YYYY"))
     }
 
-    "validate should not return any error when the given date is valid" in {
+    "validate should not return any error when the given date is valid and future date is allowed" in {
```

Comment:
I think the test should remain how it was and instead, you add a new test to the bottom of the file ("validate should not return an error when future date is allowed and the given date is in future") because this test was originally testing whether the date is entered correctly i.e. what the user entered was an official date; adding the future date part on top of it makes it test for two things.

Reply:
Yeah, that makes sense. I added another test, thanks